### PR TITLE
Unit tests & linting

### PR DESCRIPTION
**Pull request by kvikshaug**
This installs testing tools and makes sure tests & linting is run on CI. The actual test suite provides a bare minimum, ensuring that all components are mountable. This should at least provide a good foundation on which to build the test suite. In the future, e2e tests with [cypress](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-e2e-cypress) or [nightwatch](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-e2e-nightwatch) could be considered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,53 @@
-# memote-service-frontend
+# Memote
 
-## Project setup
+This is the frontend client for the memote.io web site, a free online service providing access to memote, the metabolic model test suite. See also the [backend service implementation](https://github.com/dd-decaf/memote-webservice).
+
+For more information about memote, see [opencobra/memote](https://github.com/opencobra/memote).
+
+## Technology stack
```
Comment (ChristianLieven) [inline README.md]: Love this!

File: src/App.vue [hunk 2]
```diff
@@ -16,12 +16,15 @@ import Welcome from './components/Welcome.vue';
 import Footer from './components/Footer.vue';
 
 export default {
-  name: 'app',
   components: {
+    // See: https://github.com/vuejs/eslint-plugin-vue/issues/556
+    // eslint-disable-next-line vue/no-unused-components
```
Comment (ChristianLieven) [inline src/App.vue]: So despite this issue still existing the Vue Devs closed the Issue?

File: tests/unit/App.spec.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+localVue.use(Vuex);
+
+describe('App.vue', () => {
+  it('mounts and creates the app container', () => {
+    const store = new Vuex.Store({
+      state: { tasks: [] },
+      actions: { readTasks() {} },
+    });
+    const wrapper = shallowMount(App, { store, localVue });
+    expect(wrapper.find('#app').exists()).to.be.true;
```
Comment (ChristianLieven) [inline tests/unit/App.spec.js]: This seems so arcane to me: `to.be.true`. Welp.

File: src/App.vue (hunk 2 above)
Comment (kvikshaug) [reply src/App.vue]: It's fixed but released in the next version

File: tests/unit/App.spec.js (hunk 3 above)
Comment (kvikshaug) [reply tests/unit/App.spec.js]: This is the API for behaviour driven tests in chai, there are some good explanations in the [introduction](https://www.chaijs.com/guide/styles/#expect) and [api docs](https://www.chaijs.com/api/bdd/)

**Pull request merged**